src/background/tab-state.js

```diff
@@ -86,6 +94,7 @@ export default function TabState(initialState, onchange) {
 
   this.clearTab = function (tabId) {
     this.setState(tabId, null);
+    pendingAnnotationCountRequests.delete(tabId);
```

Comment:
This makes me realize that it would may make sense to store the pending request in the per-tab state. However this would require a bunch of other changes and refactoring so I suggest we should not try to do that in this PR.

Reply:
OK